egui/src/paint/font.rs

```diff
@@ -426,7 +426,10 @@ impl Font {
             }
 
             const NON_BREAKING_SPACE: char = '\u{A0}';
-            if chr.is_whitespace() && chr != NON_BREAKING_SPACE {
+            let is_ch = |c|{
+                (c >= '\u{4E00}' && c <= '\u{9FA5}') || (c >= '\u{3400}' && c <= '\u{4DB5}') //Chinese
+            };
```

Comment:
Please make this a free function:

```
fn is_chinese(c: char) -> bool { … }
```

Reply:
Ok, it is done. P.S. like Chinese, Japanese and Korean are both hieroglyphs, so the newline rule may be the same. But I am not familliar with either of them. Maybe it's better to leave them to some experts.